Description: Ensure BANKDATA_PATH is set
 Otherwise the ktoblzcheck binary wouldn't work without providing a bank data
 file. This was found by running the autopkgtest.
Author: Micha Lenk <micha@debian.org>
Origin: Debian
Forwarded: no
Last-Update: 2019-10-06

--- libktoblzcheck-1.50.orig/CMakeLists.txt
+++ libktoblzcheck-1.50/CMakeLists.txt
@@ -137,7 +137,7 @@ set(bindir "\${prefix}/${CMAKE_INSTALL_B
 set(libdir "\${prefix}/${CMAKE_INSTALL_LIBDIR}")
 set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
 set(datadir "\${prefix}/${CMAKE_INSTALL_DATADIR}")
-set(BANKDATA_PATH "\${pkgdatadir}")
+set(BANKDATA_PATH "${CMAKE_INSTALL_FULL_DATADIR}/ktoblzcheck")
 if (Python2_Development_FOUND)
     set(pythondir "\${prefix}/${Python2_SITEARCH}")
 endif()
--- libktoblzcheck-1.50.orig/config.h.cmake
+++ libktoblzcheck-1.50/config.h.cmake
@@ -56,4 +56,6 @@
 /* Define to 1 if you have the ANSI C header files. */
 #cmakedefine STDC_HEADERS "@STDC_HEADERS@"
 /* Version number of package */
-#cmakedefine VERSION "@VERSION@"
\ No newline at end of file
+#cmakedefine VERSION "@VERSION@"
+/* Location of bank data file */
+#cmakedefine BANKDATA_PATH "@BANKDATA_PATH@"
